dm/syncer/expr_filter_group.go

```diff
@@ -200,13 +203,13 @@ func SkipDMLByExpression(ctx sessionctx.Context, row []interface{}, expr express
 }
 
 // getSimpleExprOfTable returns an expression of given `expr` string, using the table structure that is tracked before.
-func getSimpleExprOfTable(ctx sessionctx.Context, expr string, ti *model.TableInfo) (expression.Expression, error) {
+func getSimpleExprOfTable(ctx sessionctx.Context, expr string, ti *model.TableInfo, logger log.Logger) (expression.Expression, error) {
```

Comment:
I notice `tidbcontext` can put the logger in context, related to https://github.com/pingcap/tidb/blob/master/util/logutil/log.go#L184. 
Does it help?

Reply:
this is sessionctx.Contex, not context